src/components/Accordion/Accordion.module.css

```diff
@@ -0,0 +1,84 @@
+.accordionItem:first-child .accordionTrigger {
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+}
+
+.accordionItem[data-state="closed"]:last-child .accordionTrigger,
+.accordionItem[data-state="open"]:last-child .content {
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
+}
+
+.accordionTrigger {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 40px;
+  padding: 0 16px;
+  margin-top: -1px;
+  box-sizing: border-box;
+  border: 1px solid var(--neutral-6);
+  color: var(--neutral-12);
+  z-index: 0;
+}
+
+.accordionTrigger:hover,
+.accordionTrigger:focus-visible {
+  background-color: var(--brand-2);
+  border-color: var(--brand-8);
+  z-index: 1;
+}
+
+.accordionTrigger:focus-visible {
+  outline: 1px solid var(--brand-7);
+}
+
+.accordionTriggerTitle {
+  font-size: var(--font-size-x-small);
+}
+
+.accordionTrigger[data-state="open"] .plus {
+  display: none;
+}
+
+.accordionTrigger[data-state="closed"] .minus {
+  display: none;
+}
+
+.accordionContent {
+  overflow: hidden;
+}
+
+.accordionContent[data-state="open"] {
+  animation: slideDown 300ms ease-out;
+}
+
+.accordionContent[data-state="closed"] {
+  animation: slideUp 300ms ease-out;
+}
```

Comment:
Minor nitpicking: It might be a personal preference, but I tried changing the animation time to 200ms and thought it felt smoother (a bit faster). Try it, what do you think? :) If you think it feels better with 300ms, keep it :D

Reply:
Yes good idea, thanks! :)